pydantic_mongo/abstract_repository.py

```diff
@@ -70,7 +70,7 @@ def to_document(model: T) -> dict:
         :return: dict
         """
         model_with_id = cast(ModelWithId, model)
-        data = model_with_id.model_dump()
+        data = model_with_id.model_dump(mode="json")
```

Comment:
Perhaps we should exclude  [computed_field](https://docs.pydantic.dev/latest/concepts/fields/#the-computed_field-decorator)  here

This can be done for example like this:
```python
computed_fields = set(model.model_computed_fields)
return model.model_dump(
    mode="json", 
    exclude=computed_fields,
 )
```